samples/bluetooth/hci_usb/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _bluetooth-hci-usb-sample:
+.. zephyr:code-sample:: bluetooth_hci_usb
+   :name: HCI USB
+   :relevant-api: hci_raw bluetooth _usb_device_core_api usbd_api
 
-Bluetooth: HCI USB
-##################
+   Turn a Zephyr board into a USB Bluetooth dongle.
```

Comment:
Turn a Zephyr board into a USB Bluetooth dongle. Compatible with all major operating systems (Mac, Windows, Linux).

Reply:
Not sure I agree with making the "short" description that long but I get your point and will see if I can add the precision in a maybe more concise way if you really think it's important to have it right at the sample listing level.